engine/src/main/java/org/hibernate/validator/internal/engine/validationcontext/AbstractValidationContext.java

```diff
@@ -288,7 +288,7 @@ public AbstractConstraintValidatorContext createConstraintValidatorContextFor(Co
 	@Override
 	public String toString() {
 		final StringBuilder sb = new StringBuilder();
-		sb.append( "ValidationContext" );
+		sb.append( "BaseBeanValidationContext" );
```

Comment:
Not very excited about this toString() here. Let's at least use the concrete class name. And tbh, I would have implemented the `toString()` in each child so that I can add the proper context information.